src/Game/UI/Controls/ItemGump.cs

```diff
@@ -314,11 +314,23 @@ protected override bool OnMouseDoubleClick(int x, int y, MouseButton button)
         {
             if (button != MouseButton.Left)
                 return false;
-
-            GameActions.DoubleClick(LocalSerial);
+ 
+            Item item, container;
+ 
+            if (
+                Engine.Profile.Current.DoubleClickToLootInsideContainers &&
+                (item = World.Items.Get(LocalSerial)) != null &&
+                item.ItemData.IsContainer == false &&
```

Comment:
2 things:
- style: item.ItemData.IsContainer == false    ---->    !item.ItemData.IsContainer
- game behaviour: !item.ItemData.IsContainer && item.Items.Count == 0   [just to be sure that it is not a container, sometimes custom uo files missing these infos]

Reply:
Fixed